src/lib/components/workflow/workflow-relationships.svelte

```diff
@@ -2,38 +2,27 @@
   import { page } from '$app/stores';
   import { routeForEventHistory } from '$lib/utilities/route-for';
   import { workflowRun } from '$lib/stores/workflow-run';
-  import { eventHistory } from '$lib/stores/events';
 
   import Accordion from '$lib/holocene/accordion.svelte';
   import Badge from '$lib/holocene/badge.svelte';
   import ChildWorkflowsTable from '$lib/components/workflow/child-workflows-table.svelte';
   import WorkflowDetail from '$lib/components/workflow/workflow-detail.svelte';
-  import type { CombinedAttributes } from '$lib/utilities/format-event-attributes';
 
-  type WorkflowEventWithAttributes = WorkflowEvent & {
-    attributes: CombinedAttributes;
-  };
+  export let hasChildren: boolean;
+  export let hasRelationships: boolean;
+  export let first: string;
+  export let parent: WorkflowIdentifier;
+  export let next: string;
+  export let previous: string;
 
-  $: children = $workflowRun.workflow?.pendingChildren.length;
-  $: parent = $workflowRun.workflow?.parent;
-  $: lastEvent = $eventHistory.end[0] as WorkflowEventWithAttributes;
-  $: firstEvent = $eventHistory.start[0] as WorkflowEventWithAttributes;
-  $: firstExecutionRunId = firstEvent?.attributes?.firstExecutionRunId;
-  $: first =
-    firstExecutionRunId === $workflowRun.workflow?.runId
-      ? undefined // don't show first if it is the same as the current workflow run ID
-      : firstExecutionRunId;
-  $: previous = firstEvent?.attributes?.continuedExecutionRunId;
-  $: next = lastEvent?.attributes?.newExecutionRunId;
-  $: hasRelationships = parent || children || first || previous || next;
   $: ({ workflow, namespace } = $page.params);
 </script>
 
 <Accordion title="Relationships" icon="relationship">
   <div slot="summary" class="hidden flex-row gap-2 lg:flex">
     <Badge type={parent ? 'purple' : 'gray'}>{parent ? 1 : 0} Parent</Badge>
-    <Badge type={children ? 'purple' : 'gray'}
-      >{children} Pending Children</Badge
+    <Badge type={hasChildren ? 'purple' : 'gray'}
+      >{hasChildren} Pending Children</Badge
```

Comment:
Maybe we want a `numOfChildren` variable instead? Otherwise we should update this to 👇 
```suggestion
      >{$workflowRun.workflow.pendingChildren.length} Pending Children</Badge
```
so we have a number here 

<img width="815" alt="Screenshot 2023-01-26 at 2 37 34 PM" src="https://user-images.githubusercontent.com/15069288/214956229-c6df4e81-8ab0-41b1-b438-af97c120ac1a.png">